rotkehlchen/db/updates.py

```diff
@@ -7,7 +7,7 @@
 import requests
 
 from rotkehlchen.assets.spam_assets import update_spam_assets
-from rotkehlchen.constants.timing import DEFAULT_TIMEOUT_TUPLE
+from rotkehlchen.db import settings
```

Comment:
Don't do that. Where in the code have you ever seen us import entire modules like this. Import only what you need.
```suggestion
from rotkehlchen.db.settings import CachedSettings
```

Reply:
I do the import like this, to solve circular dependency issues. If you do  "from rotkehlchen.db.settings import CachedSettings" then you get circular dependency error. If I'm not mistaken the reason is that in the latter case, python evaluate the whole module on import (runs the top level code etc.) to find CachedSettings and this causes the circular dependency..